.github/workflows/e2e-parallel-full.yml

```diff
@@ -86,4 +107,42 @@ jobs:
         run: |
           echo "Terraform Apply step failed...Please check the logs of the Terraform Apply step."
           echo "Failing the job to avoid false positives."
+          kill ${{ env.iamlive_pid }}
+          while $(kill -0 ${{ env.iamlive_pid }} 2>/dev/null); do sleep 1; done;
+          cat ${HOME}/policy.json
           exit 1
+
+      - name: Iamlive Print & Upload Policy
+        if: ${{ always() }}
+        run: |
+          kill ${{ env.iamlive_pid }}
+          while $(kill -0 ${{ env.iamlive_pid }} 2>/dev/null); do sleep 1; done;
+          cat ${HOME}/policy.json
+          aws s3 cp ${HOME}/policy.json s3://eks-blueprints-iam-policies/${{ matrix.example_path }}.json
```

Comment:
1. We should make the bucket name either a workflow global env variable or a github secret to ensure its defined in one place correctly (is it `terraform-eks-blueprints-iam-policies-examples` or `eks-blueprints-iam-policies` ?)
2. do we need to add in a git ref or something so that PRs aren't clobbering one another? adding something like `/${{ github.ref }}/` into the prefix?
3. If we do that, do we need/want to set a lifecycle policy on these